### PR TITLE
Article: Centralized authentication/SSO 

### PR DESCRIPTION
**Pull request by stevenrombauts**
Adds the article on how to configure centralized authentication/SSO. 

Trello: https://trello.com/c/5xONIAAL/4391-document-how-to-use-microsoft-entra-id-okta-other-providers-namely-through-code-hosting-sync-oauth-enforcement-settings

### CODE REVIEW

File: docs/centralized-authentication-and-sso.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+If you set up your own integration in Step 1, the final step is sharing your organization-specific login link with your users. 
+Look for the "Login link" button in _Settings > Integrations_ page.
+
+![Login link settings](/Resources/public/img/docs/articles/centralized-authentication-and-sso/login-link-settings.png)
+
+This link presents the option to authenticate with your configured integration. Once the user clicks on the
+"Login with &lt;integration name&gt;" button, the authentication process is initiated and your users are redirected to your 
+code-hosting platform to log in. . 
+
+![Login link example](/Resources/public/img/docs/articles/centralized-authentication-and-sso/login-link-example.png)
```
Comment (glaubinix) [inline docs/centralized-authentication-and-sso.md:83]: The link says "Log in with Self-Hosted Gitlab" but the screenshot before says "GitLab" is this a bug or issue with the screenshot?

File: docs/centralized-authentication-and-sso.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+### 5 - Share the organization-specific login link
+
+If you set up your own integration in Step 1, the final step is sharing your organization-specific login link with your users. 
+Look for the "Login link" button in _Settings > Integrations_ page.
+
+![Login link settings](/Resources/public/img/docs/articles/centralized-authentication-and-sso/login-link-settings.png)
+
+This link presents the option to authenticate with your configured integration. Once the user clicks on the
+"Login with &lt;integration name&gt;" button, the authentication process is initiated and your users are redirected to your
```
Comment (glaubinix) [inline docs/centralized-authentication-and-sso.md:80]:
```suggestion
"Log in with &lt;integration name&gt;" button, the authentication process is initiated and your users are redirected to your 
```

File: docs/centralized-authentication-and-sso.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+### 5 - Share the organization-specific login link
+
+If you set up your own integration in Step 1, the final step is sharing your organization-specific login link with your users. 
+Look for the "Login link" button in _Settings > Integrations_ page.
+
+![Login link settings](/Resources/public/img/docs/articles/centralized-authentication-and-sso/login-link-settings.png)
+
+This link presents the option to authenticate with your configured integration. Once the user clicks on the
+"Login with &lt;integration name&gt;" button, the authentication process is initiated and your users are redirected to your 
+code-hosting platform to log in. .
```
Comment (glaubinix) [inline docs/centralized-authentication-and-sso.md:81]:
```suggestion
code-hosting platform to log in.
```

File: docs/centralized-authentication-and-sso.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+![Login link settings](/Resources/public/img/docs/articles/centralized-authentication-and-sso/login-link-settings.png)
+
+This link presents the option to authenticate with your configured integration. Once the user clicks on the
+"Login with &lt;integration name&gt;" button, the authentication process is initiated and your users are redirected to your 
+code-hosting platform to log in. . 
+
+![Login link example](/Resources/public/img/docs/articles/centralized-authentication-and-sso/login-link-example.png)
+
+If you are using one of the default platforms on Private Packagist Cloud, there is no
+need for a custom login link
```
Comment (glaubinix) [inline docs/centralized-authentication-and-sso.md:86]:
```suggestion
If you are using one of the default platforms on Private Packagist Cloud, there is no
need for a custom login link.
```

File: docs/centralized-authentication-and-sso.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+flow, and associate your Private Packagist account with your GitHub, GitLab or Bitbucket account.
+
+![Connect your account via OAuth](/Resources/public/img/docs/articles/centralized-authentication-and-sso/profile-connect.png)
+
+### 3 - Set up the synchronization
+
+Now we can continue to set up the synchronization. In your organization, go to _Settings > Synchronizations_ and press the 
+"Add Synchronization" button. Follow the steps to complete the configuration.
+
+![Create synchronization](/Resources/public/img/docs/articles/centralized-authentication-and-sso/configure-synchronization.png)
```
Comment (glaubinix) [inline docs/centralized-authentication-and-sso.md:46]:
```suggestion
![Create a synchronization](/Resources/public/img/docs/articles/centralized-authentication-and-sso/configure-synchronization.png)
```

File: docs/centralized-authentication-and-sso.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+once they log in via OAuth. Only users who have logged into Private Packagist using OAuth will be considered active and count for billing purposes.
+
+Whether or not you are using automatic user provisioning, Private Packagist will always manage the same users as those provisioned 
+in your code hosting platform. If a user is removed, that same user will also be removed from Private Packagist.
+
+If you want to control which of your users should be allowed to access Private Packagist at all, you can opt to deactivate 
+users discovered by the synchronization by default. Then, only users that you activate manually will be able to log in. 
+To enable this behavior, go to the _Teams_ tab, and click on "View and manage all members of this organization" and check the box:
+
+![Deactivate users discovered by sync](/Resources/public/img/docs/articles/centralized-authentication-and-sso/deactivate-sync-members.png)
```
Comment (glaubinix) [inline docs/centralized-authentication-and-sso.md:59]:
```suggestion
![Deactivate users discovered by synchronization](/Resources/public/img/docs/articles/centralized-authentication-and-sso/deactivate-sync-members.png)
```

**Review by glaubinix — CHANGES_REQUESTED** [review]

File: docs/centralized-authentication-and-sso.md (hunk 1 above)
Comment (stevenrombauts) [reply docs/centralized-authentication-and-sso.md:83]: Weird. That's not what I'm seeing here: 

CleanShot 2024-11-26 at 13 56 02

Comment (glaubinix) [reply docs/centralized-authentication-and-sso.md:83]: The text in the red circle is different. One says `GitLab` the other `Gitlab`

Comment (stevenrombauts) [reply docs/centralized-authentication-and-sso.md:83]: Oh! I thought you meant "self-hosted" text was missing. Screenshot must be from two different attempts. Thanks!

**Pull request merged**